content/en/dora_metrics/setup/failures.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Understanding Incident Data for DORA Metrics
+description: Learn how to send incident events for DORA Metrics.
+aliases:
+- /continuous_integration/dora_metrics/setup/incidents
+- /dora_metrics/setup/incidents
+- /dora_metrics/failures/incident_api
+- /dora_metrics/failures/pagerduty
+- /dora_metrics/failures/
+is_beta: true
+further_reading:
+- link: "/continuous_integration/dora_metrics/setup/deployments"
+  tag: "Documentation"
+  text: "Learn about setting up deployment data in DORA Metrics"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "https://github.com/DataDog/datadog-ci"
+  tag: "Source Code"
+  text: "Learn about the datadog-ci CLI tool"
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+- link: "https://app.datadoghq.com/release-notes?category=Software%20Delivery"
+  tag: "Release Notes"
+  text: "Check out the latest Software Delivery releases! (App login required)"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">DORA Metrics is in public beta.</div>
+
+## Overview
+
+Failed deployments events, currently interpreted through incident events, are used to compute [change failure rate](#calculating-change-failure-rate) and [mean time to restore (MTTR)](#calculating-mean-time-to-restore). 
+
+## Configuring an incident data source
```

Comment:
I would keep the word "selecting" because the customer cannot configure the two data sources. They have to select one of them + configuring it

So we can put `Selecting` or `Selecting and configuring`

Reply:
```suggestion
## Selecting and configuring an incident data source
```